From 5f907d27b72ae942e0a929e685c75bd752103527 Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sun, 18 Oct 2020 18:05:54 +0200
Subject: [PATCH 100/106] Revert "drm/vmwgfx: check master authentication in
 surface_ref ioctls"

This reverts commit 4872e6aa217fbb475ffa0ad7bda0d9acff543f2c.
---
 drivers/gpu/drm/vmwgfx/vmwgfx_surface.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/vmwgfx/vmwgfx_surface.c b/drivers/gpu/drm/vmwgfx/vmwgfx_surface.c
index 3914bfee0533..6583aae99c0a 100644
--- a/drivers/gpu/drm/vmwgfx/vmwgfx_surface.c
+++ b/drivers/gpu/drm/vmwgfx/vmwgfx_surface.c
@@ -965,13 +965,6 @@ vmw_surface_handle_reference(struct vmw_private *dev_priv,
 		user_srf = container_of(base, struct vmw_user_surface,
 					prime.base);
 
-		/* Error out if we are unauthenticated primary */
-		if (drm_is_primary_client(file_priv) &&
-		    !file_priv->authenticated) {
-			ret = -EACCES;
-			goto out_bad_resource;
-		}
-
 		/*
 		 * Make sure the surface creator has the same
 		 * authenticating master, or is already registered with us.
-- 
2.17.1

